resources/quality/tizyx/tizyx_evy/tizyx_evy_global_High_Quality.inst.cfg

```diff
@@ -0,0 +1,31 @@
+[general]
+version = 4
+name = High
+definition = tizyx_evy
+
+[metadata]
+setting_version = 5
+type = quality
+quality_type = high
+weight = 1
+global_quality = True
+
+[values]
+layer_height = 0.1
+layer_height_0 = 0.1
+
+adhesion_type = skirt
+skirt_line_count = 2
+skirt_gap = 2
+fill_outline_gaps = True
+infill_sparse_density = 15
+material_diameter = 1.75
```

Comment:
Don't need to define this here, since the value is already set in the extruder definition.